packages/xo-web/src/common/selectors.js

```diff
@@ -270,7 +272,11 @@ const _getPermissionsPredicate = invoke(() => {
     }
   )
 
-  return state => {
+  return (state, props) => {
+    if (props !== undefined && props.resourceSets !== undefined) {
```

Comment:
Can it be abused in some way? If there is a `resourceSets` prop passed to a component, then all the objects can be accessed in that component, no matter the user's permissions. It feels a bit dangerous.